File: Upcoder/Other/LTCB_GK_cau2/LTCB_GK_cau2.cpp
#include<bits/stdc++.h>
using namespace std;
int main()
{
    int n;
    cin >> n;
    if(n % 3 == 0)
        cout << n << " chia het cho 3";
    if(n % 3 == 2)
        cout << n << " chia 3 du " << n % 3;
    if(n % 3 == 1)
        cout << n << " chia 3 du " << n % 3;
}

File: Upcoder/Other/LTCB_GK_cau2/LTCB_GK_cau2.py
n = int(input())

if n % 3 == 0:
    print("{} chia het cho 3".format(n), end="")
elif n % 3 == 1:
    print("{} chia 3 du {}".format(n, n % 3), end="")
else:  # n % 3 == 2
    print("{} chia 3 du {}".format(n, n % 3), end="")
